<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>titlebar</class>
 <widget class="QWidget" name="titlebar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>65</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1268</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>108</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="icon_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>68</width>
       <height>68</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>68</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="framelesswindow.qrc">:/frameless_images/logo.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="titleText_lable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">  padding-left:5px;
  color:rgb(153,153,153);</string>
         </property>
         <property name="text">
          <string>titleText_lable</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="noteButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#noteButton{
  background-color:none;
  border:none;
  width:80px;
  height:20px;
  padding:4px;
}
#noteButton:hover{
  background-color:palette(alternate-base);
}
#noteButton:pressed{
  background-color:palette(highlight);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="framelesswindow.qrc">
           <normaloff>:/frameless_images/note_button.png</normaloff>:/frameless_images/note_button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="skinButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">#skinButton{
  background-color:none;
  border:none;
  width:30px;
  height:30px;
  padding:4px;
  image:url(:/frameless_images/skin_button.png);
}
#skinButton:hover{
  background-color:palette(alternate-base);
}
#skinButton:pressed{
  background-color:palette(highlight);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="minimizeButton">
         <property name="styleSheet">
          <string notr="true">#minimizeButton{
  background-color:none;
  border:none;
  width:16px;
  height:16px;
  padding:4px;
  image:url(:/frameless_images/icon_window_minimize.png);
}
#minimizeButton:hover{
  background-color:palette(alternate-base);
}
#minimizeButton:pressed{
  background-color:palette(highlight);
}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/icon_window_minimize.png</normaloff>:/images/icon_window_minimize.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="restoreButton">
         <property name="styleSheet">
          <string notr="true">#restoreButton{
  background-color:none;
  border:none;
  width:16px;
  height:16px;
  padding:4px;
  image:url(:/frameless_images/icon_window_restore.png);
}
#restoreButton:hover{
  background-color:palette(alternate-base);
}
#restoreButton:pressed{
  background-color:palette(highlight);
}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/icon_window_restore.png</normaloff>:/images/icon_window_restore.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="maximizeButton">
         <property name="styleSheet">
          <string notr="true">#maximizeButton{
  background-color:none;
  border:none;
  width:16px;
  height:16px;
  padding:4px;
  image:url(:/frameless_images/icon_window_maximize.png);
}
#maximizeButton:hover{
  background-color:palette(alternate-base);
}
##maximizeButton:pressed{
  background-color:palette(highlight);
}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/icon_window_maximize.png</normaloff>:/images/icon_window_maximize.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="closeButton">
         <property name="styleSheet">
          <string notr="true">#closeButton{
  background-color:none;
  border:none;
  width:16px;
  height:16px;
  padding:4px;
  image:url(:/frameless_images/icon_window_close.png);
  border-top-right-radius: 5px;
}
#closeButton:hover{
  background-color:palette(alternate-base);
}
##closeButton:pressed{
  background-color:palette(highlight);
}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/icon_window_close.png</normaloff>:/images/icon_window_close.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="darkstyle_action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>darkstyle</string>
   </property>
  </action>
  <action name="lightstyle_action">
   <property name="text">
    <string>lightstyle</string>
   </property>
  </action>
  <action name="ubuntustyle_action">
   <property name="text">
    <string>ubuntustyle</string>
   </property>
  </action>
  <action name="macosstyle_action">
   <property name="text">
    <string>macosstyle</string>
   </property>
  </action>
  <action name="defaultstyle_action">
   <property name="text">
    <string>defaultstyle</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="framelesswindow.qrc"/>
 </resources>
 <connections/>
</ui>
